src/api/app/controllers/webui/package_controller.rb

```diff
@@ -2,6 +2,7 @@ class Webui::PackageController < Webui::WebuiController
   include ParsePackageDiff
   include Webui::PackageHelper
   include Webui::ManageRelationships
+  include Webui::Notifications
```

Comment:
What about renaming it to `Webui::NotificationsHandler`, I find it a bit more descriptive.

Reply:
I like it. :+1: Changed.